lab1.js

```diff
@@ -134,6 +163,17 @@ var tooHungryDay;
 */
 
 // your code goes here
+function lionMeals() {
+  var totalMealsPerDay = 0;
+  var avgMealsPerDay = 0;
+  for (var i = 0; i < mealsPerDay.length; i++) {
+    totalMealsPerDay += mealsPerDay[i];
+    avgMealsPerDay = totalMealsPerDay / mealsPerDay.length;
+  }
+  return avgMealsPerDay;
```

Comment:
before you return, you need some way of deciding when the average has fallen under 4, and then assigning that day # to the variable tooHungryDay.